test/unit/spec/signaling/v2/iceconnectionmonitor.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const assert = require('assert');
+const sinon = require('sinon');
+
+const IceConnectionMonitor = require('../../../../../lib/signaling/v2/iceconnectionmonitor');
+
+describe('IceConnectionMonitor', () => {
+
+  describe('.constructor', () => {
```

Comment:
@makarandp0 Nitpick: Remove `.`?

Reply:
@makarandp0 I don't think we need to test for internal variables/methods unless absolutely required. Ideally we should just test public facing variables/methods like `start()` and `stop()`.